faq/prooffacehelp.php

```diff
@@ -752,20 +804,23 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 </p>
 
 <h5>Shortcuts to insert common tags</h5>
+<p>
+    These shortcuts may not work for everyone.
```

Comment:
... for everyone, and the Alt may be a different key combination depending on your Operating System and browser.

Reply:
Ditto, please coordinate with @lhamilton1.

Reply:
Updated.